### PR TITLE
Revert "chore(docs): update routing docs"

### PR DESCRIPTION
**Pull request by tpmccallum**
Reverts fermyon/developer#288

Just noticed that these changes are for a future version, which is yet to be released.

### CODE REVIEW

**Comment by tpmccallum** [comment]
Hi @fibonacci1729 
I'm sorry, I prematurely merged these doc updates; I was not aware they were Spin 1.0 onwards only.
I have reverted the original PR (hence this new PR) and added a new label (in this PR) called `hold_until_spin_1_0_0_release.
If you agree with this please approve and I will be sure to merge further down the track; when 1.0 is officially released.

**Comment by fibonacci1729** [comment]
No worries @tpmccallum! My mistake I meant to apply a label to indicate as such. Sal'good!

**Comment by itowlson** [comment]
Nit: I think the routing change is in Spin 0.9 rather than waiting until 1.0.  (And presumably the 'hold' label should apply to the change rather than the revert?)

File: content/spin/http-trigger.md [hunk 1]
```diff
@@ -64,11 +64,8 @@ which means the component will be invoked for every request starting with the
...
 If multiple components could potentially handle the same request based on their
-defined routes, the component whose route has the longest matching prefix 
-takes precedence. 
-
-In the following example, any request starting with the  `/foo/` prefix (e.g. `/foo/bar`)
-will be handled by `component-1`:
+defined routes, the last component defined in `spin.toml` takes precedence.
+In the following example:
 
 ```toml
```
Comment (tpmccallum) [inline content/spin/http-trigger.md:70]:
```suggestion
<!-- @nocpy -->

```toml
```

**Review by tpmccallum — COMMENTED** [review]
LGTM

**Comment by radu-matei** [comment]
If I understand correctly, the behavior shipped in 0.9 is the one described currently, and this PR is no longer needed, right?

**Pull request closed without merging**